static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -151,13 +111,33 @@ const modeText = {
 
 export default function ResourcePickerDialog(props: Props): JSX.Element {
   const { mode, isOpen, closeDialog, insertEmbed, contentNames } = props
+  const website = useWebsite()
+  const definedCategories = useMemo(() => {
+    const contentCollections =
+      website.starter?.config?.collections.filter(
+        entry => entry.category === "Content"
+      ) ?? []
+    return keyBy(contentCollections, collection => collection.name)
+  }, [website.starter?.config?.collections])
   const tabs = useMemo(
     () =>
       contentNames
-        .flatMap(name => TABS[name])
+        .flatMap(name => {
+          if (name && name === "resource") {
+            return [documentTab, videoTab, imageTab, otherFilesTab]
+          } else if (definedCategories && _.has(definedCategories, name)) {
```

Comment:
```suggestion
          } else if (_.has(definedCategories, name)) {
```
`definedCategories` might be empty, but it's never falsy. (Empty objects/arrays in js are truthy, unlike empty dictionaries/lists in python)

Reply:
Updated in a1f7c9758490ba9de2953f293f058057e134db9f.